.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,45 @@
+name: CI/CD
+
+on:
+    push:
+        branches:
+            - main
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Use Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20.x
+
+            - run: npm install
+
+            - run: npm run test
```

Comment:
Since `npm run test` ends with `exit 1`, will including this step cause the pipeline to fail and block the release stage? If so, I'm fine excluding the test step since we don't have any meaningful tests for this library.

Reply:
Removed